### PR TITLE
Added Pull Request / Developer guide lines Information

### PR DESCRIPTION
**Pull request by Numenter**
# Description
Extended CONTRIBUTING.md with Pull Request / Developer guide lines Information

Related to #35

## Type of change
- [X] Documentation

### CODE REVIEW

**Comment by RedstoneMedia** [comment]
I would like to reminded people, that #35 is still open and has no definite consensus yet. The rules that apply to node and folder names are fine, however the file naming convention on non GDScript files has not been definitely decided yet.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -5,6 +5,18 @@ The most meaningful are:
...
+---
+## Open a Pull Request
+If you want to add a feature or fix a bug. Pull Requests are the way to go. Try to keep them short and simple and follow the developer guide.  
+### Developer guide
+#### REQUIRED
+- Follow the [GDScript style guide](https://docs.godotengine.org/en/stable/tutorials/scripting/gdscript/gdscript_styleguide.html#gdscript-style-guide)
+- Use [Scony"s godot gdscript tollkit](https://github.com/Scony/godot-gdscript-toolkit/) to validate the Pull Request
+#### RECOMMENDED
+- Folder names are in snake_case
+- File names are in snake_case
```
Comment (RedstoneMedia) [inline CONTRIBUTING.md:17]: Should be "Script file names are in snake_case", as only this is official stated by the GDScript style guide.

**Review by RedstoneMedia — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (Numenter) [reply CONTRIBUTING.md:17]: It doesn't distinguish between files by design.
True, file names are not covered by the GDScript style guide, but this does not hinder us from creating rules for file names.

**Comment by Numenter** [comment]
Issue votings are more like opinion polls and don't have an effect on anything.
#35 is 7 days old and the last comment is 5 days old.

Hint: Look closely at CONTRIBUTING.md:15 :P

**Comment by RedstoneMedia** [comment]
I know that it is just an issue, but even if it were a PR it wouldn't be merged, since it doesn't have 75% yet (After almost a week of voting). 5/7 = ~0,7

**Pull request merged**